src/base.jl

```diff
@@ -52,6 +52,9 @@ struct PowerSystem{L <: ElectricLoad,
         time_length = timeseriescheckload(loads)
         !isa(sources.renewable, Nothing) ? timeserieschecksources(sources.renewable, time_length) : true
         !isa(sources.hydro, Nothing) ? timeserieschecksources(sources.hydro, time_length): true
+        calculate_thermal_limits!(branches,basepower)
+        check_angle_limits!(branches)
```

Comment:
there is no need to run the angle check twice.

Reply:
I dont get it, it should be in each constructor that takes a Array of Branches, just like the timeserieschecksources check